src/PostgrestQueryBuilder.ts

```diff
@@ -32,6 +32,11 @@ export default class PostgrestQueryBuilder<
     this.fetch = fetch
   }
 
+  setSchema(schema?: string): PostgrestQueryBuilder<Schema, Relation, Relationships> {
```

Comment:
How about a method like the one I proposed before here https://github.com/supabase/postgrest-js/issues/280#issuecomment-1158087393

```js
const { data, error } = await supabase
  .schema({ countries: "core", cities: "protected" })
  .from('countries')
  .select(`
    name,
    cities (
      name
    )
  `)
```

Just `schema` and it should take multiple objects. Otherwise this method will be subject to breaking changes.

Reply:
We can have a typescript type that's `string or object` right? If so, maybe we can make the string a global schema for starters.

@soedirgo WDYT?

Reply:
What is that syntax you mentioned when calling .schema()? I’m not familiar with the core or protected values

Reply:
Those are other schemas.

Reply:
Are they aliases? Is “countries” an alias for “core” and “cities” for “protected”?

Reply:
@bdotsamir `countries` and `cities` are the table names and `core` and `protected` are the schema names for those tables. This way, the client library can perform joins across different schemas.

Reply:
Ah, now I see. Thank you for clarifying. I’ll have a look soon